src/SFA.DAS.AssessorService.Application/Handlers/OrganisationStandards/OrganisationStandardVersionOptInHandler.cs

```diff
@@ -65,7 +65,7 @@ public async Task<OrganisationStandardVersion> Handle(OrganisationStandardVersio
                     OrganisationStandardId = organisationStandard.Id,
                     EffectiveFrom = request.EffectiveFrom,
                     EffectiveTo = request.EffectiveTo,
-                    DateVersionApproved = null,
+                    DateVersionApproved = DateTime.UtcNow,
```

Comment:
@ChrisJWoodcock have you updated the related unit tests?

Reply:
Good spot - I had not... I have now and made some changes to make it properly testable